exercises/kindergarten-garden/src/main/java/Plant.java

```diff
@@ -1,21 +0,0 @@
-public enum Plant {
```

Comment:
I'm pretty sure this file needs to stay here as it is needed to complete the exercise.